README.md

```diff
@@ -118,6 +118,31 @@ export class AppComponent {
 }
 ```
 
+`SortBy` - sort an array by a specific property in individual elements or non-object items
```

Comment:
I think it's worth mentioning what is the default configuration 😄

Reply:
I agree, default configuration is written in README now